txclib/commands.py

```diff
@@ -49,16 +48,17 @@ def cmd_init(argv, path_to_tx):
     save = options.save
     # if we already have a config file and we are not told to override it
     # in the args we have to ask
-    if os.path.isdir(os.path.join(path_to_tx, ".tx")):
+    if os.path.isfile(os.path.join(path_to_tx, ".tx/config")):
```

Comment:
@SofiaMargariti `os.path.join(path_to_tx, ".tx", "config")`

Reply:
(otherwise it will fail on Windows)

Reply:
in fact on line 69 it is correct: https://github.com/transifex/transifex-client/pull/180/commits/8e79188b0873408ad1633f2367fd300c26a9a452#diff-ee901b4e65e47147d51cd24ef2dadb0cR69
maybe just move that line up to here and use the `config_file` var instead?